railties/test/generators/actions_test.rb

```diff
@@ -448,6 +444,14 @@ def test_rake_with_env_option_should_run_rake_command_in_env
     end
   end
 
+  test "rake command with abort_on_failure option should raise on failure" do
+    capture(:stderr) do
```

Comment:
Should we assert on the capture output like some of the other tests do? If not, do we need to capture stderr?

Reply:
I don't think asserting on the output is strictly necessary.  We really just want to verify that it raises, with no guarantees about the content of its output.

But `capture(:stderr)` is necessary to prevent that output from leaking into the test log.